addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -561,6 +561,14 @@ $line-height-truncate: 1.25em;
             }
         }
 
+        .o_wslides_lesson_aside_list_links {
+            font-weight: 300;
+
+            li, div {
+                padding-left: 18px;
```

Comment:
Can't it be achieved with standard bootstrap classes (that are moreover responsive) ? Design team wants us to avoid custom css as much as possible and use classes, hence asking.